src/Physics/Multinucleon/EventGen/MECGenerator.cxx

```diff
@@ -888,6 +895,13 @@ void MECGenerator::SelectSuSALeptonKinematics(GHepRecord* event) const
   // This way if someone reuses this code, they are not tripped up by it.
   TMax = Enu - LepMass;
 
+  // Warn if fQ3Max value is suspect: 
+  if(fQ3Max<1.99){
```

Comment:
Same comment for this threshold here

Reply:
Same response too: there's now a better comment.